### DISCUSSION TITLE
Headers with with Outgoing Webhook?

### DISCUSSION THREAD

**Discussion by CharlesBeatty** [Budibase/budibase#2201] [Help]
I am making an API call using the outgoing webhook. But I need to add a header for authorization with a secret key. How can I specify a header in the webhook Setup?

Thanks in advance for any pointers.

**Comment by shogunpurple** [comment]
Hey @CharlesBeatty - this is actually not possible right now, but a great suggestion. I'm going to create an issue from it so that the team can look at it in due course.
